src/sync.d

```diff
@@ -1194,8 +1205,14 @@ final class SyncEngine
 							if (thisFileSize == 0){
 								// We can only upload zero size files via simpleFileUpload regardless of account type
 								// https://github.com/OneDrive/onedrive-api-docs/issues/53
-								response = onedrive.simpleUpload(path, parent.driveId, parent.id, baseName(path));
-								writeln(" done.");
+								try {
+									response = onedrive.simpleUpload(path, parent.driveId, parent.id, baseName(path));
+									writeln(" done.");
+								} catch (OneDriveException e) {
+									// error uploading file
+									return;
```

Comment:
should this be logged?  catches that don't do anything but return can be hard to debug...